expression/builtin_encryption.go

```diff
@@ -470,13 +480,23 @@ func (b *builtinEncodeSig) evalString(row chunk.Row) (string, bool, error) {
 	if isNull || err != nil {
 		return "", true, err
 	}
+	decodeTp := b.args[0].GetType()
+	decodeBuff, err := charset.NewEncoding(decodeTp.Charset).EncodeString(decodeStr)
+	if err != nil {
+		return "", true, err
```

Comment:
IMO, the return value `isNull` is for function argument. If we meet a error in function, it shouldn't return true. cc @xiongjiwei @zimulala

Reply:
Yes, I think so.

Reply:
isNull return false.